devenv/scripts/run_private_cluster.sh

```diff
@@ -0,0 +1,10 @@
+	set -e
+	echo "beginning private cluster tests..."
+	make dev CLUSTER_TYPE=private DNS_ZONE_TYPE=public
+	make push
+	make e2e
+
+  make dev CLUSTER_TYPE=private DNS_ZONE_TYPE=private
```

Comment:
why is the indentation weird?

Reply:
looks fine on ide but got rid of indentation